streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKTableLeftJoinTest.java

```diff
@@ -47,10 +45,8 @@
     private String topic1 = "topic1";
     private String topic2 = "topic2";
 
-    private IntegerSerializer keySerializer = new IntegerSerializer();
-    private StringSerializer valSerializer = new StringSerializer();
-    private IntegerDeserializer keyDeserializer = new IntegerDeserializer();
-    private StringDeserializer valDeserializer = new StringDeserializer();
+    final private Serde<Integer> keySerde = new Serdes.IntegerSerde();
```

Comment:
Nomenclature:  Sometimes we call these variables `keySerde` (= how they will be used), sometimes `intSerde` (= what they are).  Do we want to be consistent?  If so, I'd prefer `intSerde` (= what they are).